include/maliput/routing/find_lane_sequences.h

```diff
@@ -56,5 +56,11 @@ std::vector<std::vector<const maliput::api::Lane*>> FindLaneSequences(const mali
                                                                       const maliput::api::Lane* end,
                                                                       double max_length_m);
 
+/// Overload to @ref FindLaneSequences() that removes from the result those
+/// sequences presenting U-turns when @p remove_u_turns is true. Otherwise,
+/// the function behaves exactly the same.
```

Comment:
We should clarify what a u-turn means. Normally, a u-turn involves changing from one lane to an adjacent lane going in the opposite direction. In this case, we are reversing direction without actually changing lanes. We could say:

> In this context, a u-turn is when the route backtracks against itself _within the same lane_.

Reply:
Done in c908cba.